### PR TITLE
Version fix

### PR DESCRIPTION
**Pull request by nilesvm**
Changed version from module variable to nested class to get pages w version numbers to work.

### CODE REVIEW

**Comment by tshedor** [comment]
@nilesvm is this a fix for Rails 5? Does it resolve #139 ?

**Comment by tshedor** [comment]
@jamesmk did you edit my comment?

File: app/views/fae/application/_credit.slim [hunk 1]
```diff
@@ -1,5 +1,7 @@
 .credit
-  span a <a href="https://wearefine.com" target="_blank">FINE</a> Admin v#{Fae::Version.number}
+  span
+    | <a href="https://wearefine.com" target="_blank">FINE</a>
```
Comment (tshedor) [inline app/views/fae/application/_credit.slim]: This changes the markup. It prints as "a FINE Admin v###". This prints as "FINEAdmin v###"

If you guys are changing the markup to this, please Slimify this link - `a> href="https://wearefine.com" target="_blank" FINE`

**Comment by jamesmk** [comment]
@tshedor no, we're working through a couple things via slack

File: app/views/fae/application/_credit.slim (hunk 1 above)
Comment (jamesmk) [inline app/views/fae/application/_credit.slim]: yeah, this change got reverted

Comment (nilesvm) [inline app/views/fae/application/_credit.slim]: Was trying to see if slim was interpreting that a as an a tag.

**Pull request merged**